src/gluonts/model/forecast.py

```diff
@@ -516,7 +516,10 @@ class OutputType(str, Enum):
 
 
 class Config(pydantic.BaseModel):
-    num_eval_samples: int
+    class Config:
+        allow_population_by_alias = True
+
+    num_eval_samples: int = pydantic.Schema(..., alias='num_samples')
```

Comment:
Nice, I didn't know about this feature! Can we have multiple aliases (e.g., so that both `num_samples` and `num_eval samples` work correctly)?

Reply:
That's what it's doing in this case. This requires `allow_population_by_alias` to be set as well.